automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala

```diff
@@ -124,13 +124,13 @@ trait GPAllocUtils extends BillingFixtures with LeonardoTestUtils {
   }
 }
 
-trait GPAllocBeforeAndAfterAll extends GPAllocUtils with BeforeAndAfterAll {
+trait GPAllocBeforeAndAfterAll extends GPAllocUtils with BeforeAndAfterAll with LeonardoTestUtils {
```

Comment:
I'm not sure `GPAllocBeforeAndAfterAll` should extend `LeonardoTestUtils`? I think `GPAllocBeforeAndAfterAll` is intended to be mixed into the _suite_ to provide `beforeAll` and `afterAll` -- while `LeonardoTestUtils` is intended to be mixed into actual spec classes.